mongodb/trigger/mongodb/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/project-flogo/datastore-contrib/mongodb/trigger/mongodb
+
+require (
+	github.com/project-flogo/core v0.9.3
+	github.com/project-flogo/datastore-contrib/mongodb/connection v1.0.0
```

Comment:
don't think this is needed now that you are using interface, if it is needed for test, the version should probably be v0.1.0